packages/edit-post/src/components/preview-options/style.scss

```diff
@@ -0,0 +1,85 @@
+.editor-post-preview__dropdown {
+	display: none;
+	margin-right: $grid-size;
+	border: 1px solid $light-gray-600;
+}
+
+.editor-post-preview__button-toggle.editor-post-preview__button-toggle.editor-post-preview__button-toggle.editor-post-preview__button-toggle.editor-post-preview__button-toggle.editor-post-preview__button-toggle.editor-post-preview__button-toggle {
```

Comment:
Is this selector intentional? 😮

Reply:
Most definitely so 😬 
It's needed to override some [really specific default button styling](https://github.com/WordPress/gutenberg/blob/06f02c96efcf27c4d2c348ca3abe08d08ca5f51d/packages/components/src/button/style.scss#L20).

Reply:
I am revisiting that really specific styling in https://github.com/WordPress/gutenberg/pull/19348/files#diff-169503e5275f964f8bd036a7bfcd8f7cL26, and if that lands we can revisit this!